src/main/java/core/basesyntax/model/machine/Machine.java

```diff
@@ -1,9 +1,34 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Machine {
+    @Id
+    @GeneratedValue(
+            strategy = GenerationType.TABLE,
+            generator = "native_for_machine"
```

Comment:
`generator` property is used for defining custom generator
this custom generator can be described with `@GenericGenerator` annotation

here we don't need `generator` property

Reply:
Removed